### DISCUSSION TITLE
Feature Request: Switch tippy.js to Floating UI

### DISCUSSION THREAD

**Discussion by binaryfire** [ueberdosis/tiptap#4193] [Feature Requests]
### Description

Hi guys

Popper has been superseded by Floating UI (Floating UI was created by the Popper devs as a replacement for Popper): https://floating-ui.com/docs/motivation

Also tippy.js hasn't been updated in almost 2 years: https://www.npmjs.com/package/tippy.js

### Use Case

It'd be great if tippy could be switched out for Floating UI. Most modern projects are using Floating UI now, so having to add tippy.js + Popper just for tiptap adds extra bloat to projects.

### Type

Other

**Comment by vlrevolution** [comment]
I think so too. +1
It would help mitigate issue like the one seen here:
https://github.com/atomiks/tippyjs/issues/517
